### ISSUE TITLE
Define Mechanics

### ISSUE THREAD

**Issue by JamesFranc** [JamesFranc/StorybookTD#3]
Create a file rules.md that includes the following topics:
- [x] Min/Max/Recommended number of players
- [x] Average play time
- [x] Layout of the game
- [x] Order Of Play (Phases)
  - [x]  Include possible actions/counters for each phase
  - [x] Do this for both players and npcs
- [x]  Define Win conditions

**Comment by lucymariej** [comment]
Do we want to have NPCs to play against or are you just referring to the NPCs defined by the cards? Is it just for players like the physical game?

**Comment by JamesFranc** [comment]
I was referring to the NPCs defined by the cards. It is just for players like the physical game.

**Comment by lucymariej** [comment]
Cool, thanks for the clarification.

**Comment by JamesFranc** [comment]
Not a problem, let me know if you have any other questions!

**Comment by JamesFranc** [comment]
@lucymariej do we want to include a single player version? If we do, maybe we should consider a type of reward system (think: card power-up unlocks, ie: nullify next card steal, insta-kill 10+monster) to encourage users to come back and play.

Maybe we could look into incorporating a type of badge system as well, once we have an online system working.

**Comment by lucymariej** [comment]
I think a single player version would be a nice touch. How do you mean though. Do you mean a single player playing against an AI or did you have something else in mind?

In either case, I like the idea of the badge system and rewards, but I definitely think starting with just a hot seat multiplayer will work best. It might be best to start with getting the hot seat version working for 2 local players first, then we can work our way out from there. More than 2 players is going to require more mechanics.

I really like the idea of the card power-ups and, in particular, the nullify card steal boost. Would it be just a one time use? (We might want to open a separate discussion for this or table it until it becomes relevant)

**Comment by JamesFranc** [comment]
You're on to something with a separate discussion. Why don't you create a new issue and outline what you think we should discuss :8ball:

**Comment by lucymariej** [comment]
Should we include the vocabulary (as included in the original rules) as a separate file for reference. So, have a vocabulary.md file in addition to the rules?

edit: created this issue #8

**Comment by JamesFranc** [comment]
I think a vocabulary.md or maybe a JSON would be a great idea.

Thinking:

{ "vocabulary" : [
    {"term":"Apple", "definition":"a fiberous skinned fruit grown from
deciduous trees that is used as an icon in both pop culture and mythology"},
    {"term":"<name>", "definition":"<words>"},
    {"term":"<name>", "definition":"<words>"}
]}

So on and so forth ;)

Edit:
You're right we should do a .md file for now and talk more implementation once planning is done.

**Comment by lucymariej** [comment]
Rules.md is created and merged, so I'm going to go ahead and close this issue.
